website/static/js/osfHelpers.js

```diff
@@ -38,18 +39,12 @@ var growl = function(title, message, type) {
 var apiV2Url = function (pathString, paramsObject, apiPrefix){
```

Comment:
It would be worth refactoring this to use an options object rather than positional arguments because `paramsObject` should be optional and `apiPrefix` may have a default.

Also, `paramsObject` is a little ambiguous; possibly rename to `query`. Also, the "api" in "apiPrefix" is redundant.

Follow the common pattern:

```js
function apiV2URL(path, options) {
    var defaults = {
        query: {},
        prefix: '/v2/'
    }
    var opts = $.extend({}, defaults, options);
    // ...
}
```